src/dlgtrackinfo.cpp

```diff
@@ -478,6 +479,7 @@ void DlgTrackInfo::reloadTrackMetadata() {
     if (m_pLoadedTrack) {
         TrackPointer pTrack(new TrackInfoObject(m_pLoadedTrack->getLocation(),
                                                 m_pLoadedTrack->getSecurityToken()));
+        SoundSourceProxy(pTrack).loadTrackMetadata();
```

Comment:
Strange, why we cannot do just 
SoundSourceProxy(m_pLoadedTrack).loadTrackMetadata();